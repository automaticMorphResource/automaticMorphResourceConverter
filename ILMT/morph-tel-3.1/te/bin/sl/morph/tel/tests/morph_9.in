1	xAxApu	unk	
1	2	unk	
1	vela	unk	
1	kaMte	unk	
1	eVkkuva	unk	
1	rakAla	unk	
1	cepalu	unk	
1	uMde	unk	
1	I	unk	
1	rIP	unk	
1	ni	unk	
1	cUsina	unk	
1	waruvAwa	unk	
1	manaku	unk	
1	iMke	unk	
1	AkveriyAnni	unk	
1	cUdAli	unk
1	annA	unk	
1	manasu	unk	
1	oVppukoxu	unk	
1	.	unk	
