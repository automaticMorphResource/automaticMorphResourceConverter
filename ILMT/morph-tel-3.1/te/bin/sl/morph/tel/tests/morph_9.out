1	xAxApu	unk	<fs af='xAxApu,n,,sg,,d,0,0'>|<fs af='xAxApu,n,,sg,,o,ti,ti'>
2	2	unk	<fs af='2,num,,,,,,'>
3	vela	unk	<fs af='veVyyi,n,,pl,,o,ti,ti'>|<fs af='veVyyi,n,,pl,,o,ti,ti'>
4	kaMte	unk	<fs af='kaMte,avy,,,,,,'>|<fs af='kaMteV,n,,sg,,,0,0_e'>|<fs af='kaMteV,n,,sg,,,0,0_V'>|<fs af='kanu,v,any,any,any,,we,we'>|<fs af='k,n,,sg,,,aMte,aMte'>|<fs af='k,n,,sg,,,aMte,aMte'>
5	eVkkuva	unk	<fs af='eVkkuva,adj,,,,,0,0_adj'>
6	rakAla	unk	<fs af='rakaM,n,,pl,,o,ti,ti'>
7	cepalu	unk	<fs af='cepa,n,,pl,,d,0,0'>|<fs af='cepa,n,,pl,,,vu,vu'>
8	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
9	I	unk	<fs af='I,avy,,,,,,'>|<fs af='I,avy,,,,,,'>
10	rIP	unk	<fs af='rIP,n,,sg,,d,0,0'>|<fs af='rIP,n,,sg,,o,ti,ti'>
11	ni	unk	<fs af='ni,avy,,,,,0,0_avy'>
12	cUsina	unk	<fs af='cUdu,v,any,any,any,,ina,ina'>
13	waruvAwa	unk	<fs af='waruvAwa,adv,,,,,0,0_adv'>
14	manaku	unk	<fs af='manaM,pn,,pl,,,ki,ki'>|<fs af='manu,v,,sg,2,,aku,aku'>
15	iMke	unk	<fs af='iMke,avy,,,,,,'>|<fs af='iMka,avy,,,,,e,e_avy'>|<fs af='iMku,v,any,any,any,,e,e'>|<fs af='iMku,v,any,any,any,,i,i_e'>|<fs af='iMku,v,any,any,any,,e,e'>|<fs af='iMku,v,any,any,any,,i,i_e'>
16	AkveriyAnni	unk	<fs af='AkveriyaM,n,,sg,,,nu,nu'>|<fs af='AkveriyaM,n,,sg,,,ni,ni'>
17	cUdAli	unk	<fs af='cUdu,v,any,any,any,,Ali,Ali'>
18	annA	unk	<fs af='annA,avy,,,,,0,0_avy'>|<fs af='anna,n,,sg,,,0,0_A'>|<fs af='anna,n,,sg,,,0,0_V'>|<fs af='annu,n,,sg,,,0,0_A'>|<fs af='anu,v,any,any,any,,inA,inA'>
19	manasu	unk	<fs af='manasu,n,,sg,,d,0,0'>|<fs af='manasu,n,,sg,,o,ti,ti'>
20	oVppukoxu	unk	<fs af='oVppuko,v,fn,sg,3,,a,a'>
21	.	unk	<fs af='&dot;,punc,,,,,,'>
